src/GDGUkraine/templates/event.html

```diff
@@ -1,24 +1,37 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<meta charset="UTF-8">
-	<base href="/"></base>
+    <meta charset="UTF-8">
+    <base href="/"></base>
```

Comment:
Changing indentation style MUST be a separate PR, since it doesn't relate to feature. Also this makes code review more complicated. Please use same indentation style as before and move refactoring to separate PR.